presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -32,3 +32,9 @@ Map Functions
     Returns all the values in the map ``x``.
 
 See also :func:`map_agg` and :func:`multimap_agg` for creating a map as an aggregation.
+
+.. function:: map_concat(map<K,V>, map<K,V>) -> map<V>
```

Comment:
See how `map_keys` and `map_values` are written: `map_concat(x<K,V>, y<K,V>) -> map<V>`. Please also adapt the description accordingly.